src/Sets/Zonotope.jl

```diff
@@ -5,6 +5,9 @@ export Zonotope,
        scale,
        reduce_order
 
+# constructor from center and list of generators
+using LazySets.Arrays: _vector_type, _matrix_type
```

Comment:
I think the comment should have stayed below :wink:

Reply:
pfff